types/coin.go

```diff
@@ -664,13 +664,13 @@ func ParseCoin(coinStr string) (coin Coin, err error) {
 	return NewCoin(denomStr, amount), nil
 }
 
-// ParseCoins will parse out a list of coins separated by commas. If the parsing is successuful,
+// parseCoins will parse out a list of coins separated by commas. If the parsing is successuful,
 // the provided coins will be sanitized by removing zero coins and sorting the coin set. Lastly
-// a validation of the coin set is executed. If the check passes, ParseCoins will return the sanitized coins.
+// a validation of the coin set is executed. If the check passes, parseCoins will return the sanitized coins.
 // Otherwise it will return an error.
-// If an empty string is provided to ParseCoins, it returns nil Coins.
+// If an empty string is provided to parseCoins, it returns nil Coins.
 // Expected format: "{amount0}{denomination},...,{amountN}{denominationN}"
-func ParseCoins(coinsStr string) (Coins, error) {
+func parseCoins(coinsStr string) (Coins, error) {
```

Comment:
as Alessio suggested, we can remove this function (I see a deadcode lint error here)

Reply:
done